snuba/cli/multistorage_consumer.py

```diff
@@ -289,30 +257,91 @@ def handler(signum: int, frame: Any) -> None:
     processor.run()
 
 
-def build_multistorage_streaming_strategy_factory(
+LogicalTopicTuple = Tuple[
+    Topic,
+    Optional[Topic],
+    Optional[Topic],
+    Optional[Callable[[], DeadLetterQueuePolicy]],
+]
```

Comment:
Can you make this a NamedTuple or dataclass? How about calling this `ConsumerConfig` or something along those lines?